x/dualstaking/keeper/grpc_query_provider_delegators.go

```diff
@@ -0,0 +1,43 @@
+package keeper
+
+import (
+	"context"
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/lavanet/lava/utils"
+	"github.com/lavanet/lava/x/dualstaking/types"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+func (k Keeper) ProviderDelegators(goCtx context.Context, req *types.QueryProviderDelegatorsRequest) (*types.QueryProviderDelegatorsResponse, error) {
+	if req == nil {
+		return nil, status.Error(codes.InvalidArgument, "invalid request")
+	}
+
+	ctx := sdk.UnwrapSDKContext(goCtx)
+	nextEpoch, err := k.getNextEpoch(ctx)
```

Comment:
(see also CNS-525 #767 ): why next epoch?

Reply:
fixed. see https://github.com/lavanet/lava/commit/66efe1d21143fbdfaf9c436da814d43de0d4b42e
added there the `--with-pending` flag